.github/workflows/codecov.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - name: Install protoc
+        run: sudo apt-get install -y protobuf-compiler
```

Comment:
Should we lock protoc compiller version?

Reply:
Has locked ubuntu and protoc version in CI and Dockerfile.